libs/components/forms/src/lib/modules/radio/radio-group-id.service.ts

```diff
@@ -0,0 +1,62 @@
+import { Injectable } from '@angular/core';
+
+import { BehaviorSubject, Observable } from 'rxjs';
+
+interface RegisterArgs {
+  /**
+   * A unique ID for the radio component.
+   */
+  id: string;
+  /**
+   * The ID applied to the radio input element.
+   */
+  inputElementId: string;
+}
+
+/**
+ * Tracks the element IDs for all radios within a radio group.
+ * @internal
+ */
+@Injectable()
+export class SkyRadioGroupIdService {
+  public get radioIds(): Observable<string[]> {
+    return this.#radioIdsObs;
```

Comment:
should this be `#_radioIdsObs` since its used in a getter?

Reply:
It's not _exclusively_ used in a getter, so it can't use the `#_` prefix.